client/app/core/redux.service.js

```diff
@@ -0,0 +1,31 @@
+/** @ngInject */
+import { persistStore } from 'redux-persist'
+import { asyncSessionStorage } from 'redux-persist/storages'
+
+export function ReduxFactory ($rootScope, $ngRedux) {
+  const service = {
+    loadRedux: loadRedux,
+    clear: clear
+  }
+
+  return service
+  function clear () {
+    return new Promise((resolve, reject) => {
```

Comment:
imho for everything async please consider using redux-thunk or redux-saga instead.